src/lib/libwasi.js

```diff
@@ -267,6 +267,7 @@ var WasiLibrary = {
 #else
   fd_write__deps: ['$printChar'],
 #endif
+  fd_write__proxy: 'none', // Each Worker can do stdout/stderr printing on its own, without needing to proxy to the main thread.
```

Comment:
Under what circumstances is this true?    This function is used for all file isn't it, not just stdout/stderr?  Certainly we need  proxying here in a lot of cases don't we?

Reply:
Regardless, this change to the `fd_write` syscalls seems completely separate to the new `abort` proxy types.  Can we make this change separately?

And are there some existing JS libraries calls that can be marked as `abort`?

Reply:
Oh now I remember: you had asked that this PR should use the new annotations somewhere internally, and I think I had thought that logging should be specific to each Worker, so had tried to use them on the write functions when no filesystem is present. But that doesn't quite work out in the current structure.